mbed-edge-core-devmode/deb/debian/install

```diff
@@ -1,2 +1,3 @@
 build/bin/edge-core usr/bin
-../../../../mbed-edge-core-devmode/deb/debian/files/arm_update*.sh usr/lib/pelion/scripts
\ No newline at end of file
+debian/launch-edge-core.sh usr/bin
+../../../../mbed-edge-core-devmode/deb/debian/files/arm_update*.sh usr/lib/pelion/scripts
```

Comment:
can this line just be `debian/files/arm_update*.sh usr/lib/pelion/scripts` ?

Reply:
Probably should be cleaned up, although I didn't change that line.
Will submit a future PR with that cleaned up